corehq/apps/hqadmin/reports.py

```diff
@@ -266,3 +273,73 @@ def _format_date(date):
         if date:
             return parse(date).strftime(SERVER_DATETIME_FORMAT)
         return "---"
+
+
+class DeployHistoryReport(GetParamsMixin, AdminReport):
+    base_template = 'reports/base_template.html'
+
+    slug = 'deploy_history_report'
+    name = ugettext_lazy("Deploy History Report")
+
+    emailable = False
+    exportable = False
+    ajax_pagination = True
+    default_rows = 10
+
+    @property
+    def headers(self):
+        return DataTablesHeader(
+            DataTablesColumn(_("Date"), sortable=False),
+            DataTablesColumn(_("User"), sortable=False),
+            DataTablesColumn(_("Diff URL"), sortable=False),
+        )
+
+    @property
+    def rows(self):
+        deploy_list = HqDeploy.objects.all()
+        start = self.pagination.start
+        end = self.pagination.start + self.pagination.count
+        for deploy in deploy_list[start:end]:
+            yield [
+                self._format_date(deploy.date),
+                deploy.user,
+                self._hyperlink_diff_url(deploy.diff_url),
+            ]
+
+    @property
+    def total_records(self):
+        return HqDeploy.objects.count()
+
+    @cached_property
+    def _user_lookup_url(self):
+        return reverse('web_deploy_lookup')
```

Comment:
This method doesn't apply to this class - guessing it was copied over from the User List report.